src/core/buildAndValidateConfig.js

```diff
@@ -57,6 +44,23 @@ const buildAllOnInstanceConfiguredExtraParams = (
   );
 };
 
+const wrapLoggerInQueue = logger => {
```

Comment:
This wrapping doesn't correctly handle the "enabled" flag on the logger. This is fine, lets just add a comment to the logger to explain this.